MapboxNavigationUI/MGLMapView+MGLNavigationAdditions.h

```diff
@@ -0,0 +1,7 @@
+#import <Mapbox/Mapbox.h>
+
+@interface MGLMapView (MGLNavigationAdditions)
+
+- (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray *)locations;
```

Comment:
This should be exposed in [mapbox-gl-native](https://github.com/mapbox/mapbox-gl-native)